test/box-luatest/gh_7362_net_box_async_ping_test.lua

```diff
@@ -0,0 +1,40 @@
+local net = require('net.box')
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+local TIMEOUT = 5
+
+g.before_all(function()
+    g.server = server:new({alias = 'server'})
+    g.server:start()
+    g.conn = net.connect(g.server.net_box_uri)
+end)
+
+g.before_test('test_disconnected_ping', function(g)
+    g.server:stop()
```

Comment:
Maybe just do this line as a first one in `test_disconnected_ping`? Having the test preparation to be separated away so many lines above is really confusing.